cmd/checkmarxExecuteScan.go

```diff
@@ -140,7 +165,7 @@ func zipWorkspaceFiles(workspace, filterPattern string) (*os.File, error) {
 		return zipFile, errors.Wrap(err, "failed to create archive of project sources")
 	}
 	defer zipFile.Close()
-	zipFolder(workspace, zipFile, patterns)
+	_ = zipFolder(workspace, zipFile, patterns, newCheckmarxExecuteScanUtils())
```

Comment:
Having the `_ = xxx` makes it more transparent that we do not handle the return value, which is in this case an `error`. But that leads to the question: wouldn't it be better to handle the error instead of swiping it under the carpet. Are we able to continue in an reasonable way in case an error occurs? How difficult would it be for a support agent to  troubleshoot the issue without having the details from a failed `zipFolder` invocation?

Reply:
done